web/src/settings_notifications.ts

```diff
@@ -9,21 +11,30 @@ import * as confirm_dialog from "./confirm_dialog";
 import {$t, $t_html} from "./i18n";
 import * as message_notifications from "./message_notifications";
 import {page_params} from "./page_params";
+import {realm_default_settings_schema} from "./realm_user_settings_defaults";
 import * as settings_components from "./settings_components";
 import * as settings_config from "./settings_config";
+import type {SettingsPanel} from "./settings_preferences";
 import * as settings_ui from "./settings_ui";
 import {realm} from "./state_data";
 import * as stream_data from "./stream_data";
 import * as stream_settings_api from "./stream_settings_api";
 import * as stream_settings_data from "./stream_settings_data";
+import {stream_specific_notification_settings_schema} from "./stream_types";
 import * as sub_store from "./sub_store";
+import type {StreamSubscription} from "./sub_store";
 import * as ui_util from "./ui_util";
 import * as unread_ui from "./unread_ui";
-import {user_settings} from "./user_settings";
+import {
+    pm_notification_settings_schema,
+    user_settings,
+    user_settings_schema,
+} from "./user_settings";
```

Comment:
Should we just do a normal import of `user_settings` here?

Reply:
That seems reasonable.

Reply:
It's causing some line overflow that's not the nicest, I'll push it and you can let me know what you think

Reply:
Line overflow is fine.